server/leafnode_test.go

```diff
@@ -1883,3 +1883,45 @@ func TestLeafNodeNoDuplicateWithinCluster(t *testing.T) {
 		}
 	}
 }
+
+func TestLeafNodeOperatorBadCfg(t *testing.T) {
+	dir := os.TempDir()
```

Comment:
Needs something like this instead:
```
tmpDir, err := ioutil.TempDir("", "_nats-server")
if err != nil {
    t.Fatal("Could not create tmp dir")
}
defer os.RemoveAll(tmpDir)
```
right now you are making this test remove the OS temp dir, which explains the weird failures you got.

Reply:
fixed